barcampgb.org

```diff
@@ -0,0 +1 @@
+Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
```

Comment:
Yeah you need to stage and commit and push. Staging deleted files can be tricky. `git add -u` will be helpful.